### PR TITLE
AI-62: Fix Manage price sets issues

### PR DESCRIPTION
**Pull request by mukeshcompucorp**
**Before**

Price Sets

price-issue

View and Edit - Price Fields

view-price-issue

After clicking More > Disable or Enable

more

**After**

Price Sets

resolved-price-issue

View and Edit - Price Fields

resolved-view-price-issue

After clicking More > Disable or Enable

resolved-more

### CODE REVIEW

File: org.civicrm.bootstrapcivicrm/scss/civicrm/common/_tables.scss [hunk 1]
```diff
@@ -130,12 +130,18 @@
...
-      line-height: 1.1em;
+      line-height: 1.9em;
 
       div {
         padding: 0;
       }
     }
+
+    tr {
+      border-top-left-radius: 2px;
```
Comment (meladawy) [inline org.civicrm.bootstrapcivicrm/scss/civicrm/common/_tables.scss]: Can you use bootstrap varaiables for both radius?

File: org.civicrm.bootstrapcivicrm/scss/civicrm/contact/pages/_new-price-set.scss [hunk 2]
```diff
@@ -53,15 +53,65 @@
...
       @include mx-expandable-table();
       box-shadow: none !important;
+
+      th {       
+        border-bottom-color: $crm-dialog-container-border !important;
+        
+        &:first-child {
+          padding-left: 20px !important;
+        }       
+
```
Comment (meladawy) [inline org.civicrm.bootstrapcivicrm/scss/civicrm/contact/pages/_new-price-set.scss]: No need for this line

File: org.civicrm.bootstrapcivicrm/scss/civicrm/contact/pages/_new-price-set.scss [hunk 3]
```diff
@@ -53,15 +53,65 @@
...
+      }
+
+      tr td {
+        &:first-child {
+          padding-left: 20px !important;
+        }
+
```
Comment (meladawy) [inline org.civicrm.bootstrapcivicrm/scss/civicrm/contact/pages/_new-price-set.scss]: Remove this line

File: org.civicrm.bootstrapcivicrm/scss/civicrm/contact/pages/_new-price-set.scss [hunk 4]
```diff
@@ -53,15 +53,65 @@
...
+      td:last-child {
+        text-align: right;
+        padding-right: 20px;
+      }
+
+      .action-item {
+        text-transform: none;
+      }
+
```
Comment (meladawy) [inline org.civicrm.bootstrapcivicrm/scss/civicrm/contact/pages/_new-price-set.scss]: Remove this line

File: org.civicrm.bootstrapcivicrm/scss/civicrm/common/_tables.scss (hunk 1 above)
Comment (mukeshcompucorp) [inline org.civicrm.bootstrapcivicrm/scss/civicrm/common/_tables.scss]: Fixed.

File: org.civicrm.bootstrapcivicrm/scss/civicrm/contact/pages/_new-price-set.scss (hunk 2 above)
Comment (mukeshcompucorp) [inline org.civicrm.bootstrapcivicrm/scss/civicrm/contact/pages/_new-price-set.scss]: Fixed.

File: org.civicrm.bootstrapcivicrm/scss/civicrm/contact/pages/_new-price-set.scss (hunk 3 above)
Comment (mukeshcompucorp) [inline org.civicrm.bootstrapcivicrm/scss/civicrm/contact/pages/_new-price-set.scss]: Fixed.

File: org.civicrm.bootstrapcivicrm/scss/civicrm/contact/pages/_new-price-set.scss (hunk 4 above)
Comment (mukeshcompucorp) [inline org.civicrm.bootstrapcivicrm/scss/civicrm/contact/pages/_new-price-set.scss]: Fixed.

**Pull request merged**